sinks/librato.py

```diff
@@ -372,19 +372,18 @@ def flush(self):
         legacy_metrics = []
         count = 0
         
-        values = self.measurements.values()
-        
-        for measure in values:
-            tagged_metrics.append(measure)
-            count += 1
+        for v in self.measurements.values():
+            for metric in v:
+                tagged_metrics.append(metric)
+                count += 1
 
-            if count >= self.max_metrics_payload:
-                self.flush_payload(headers, tagged_metrics)
-                count = 0
-                tagged_metrics = []
+                if count >= self.max_metrics_payload:
+                    self.flush_payload(headers, tagged_metrics)
+                    count = 0
+                    tagged_metrics = []
 
         if count > 0:
-            self.flush_payload(headers, metrics)
+            self.flush_payload(headers, tagged_metrics)
```

Comment:
This was the key change here because `metrics` is actually a global variable that contains the payload from Statsite/stdin, so we were sending junk to the Librato API.